src/js/components/form-elements/v2/SelectField.jsx

```diff
@@ -82,7 +80,7 @@ const SelectField = ({
         multi={multiple}
         onKeyDown={onKeyDown}
         creatable={creatable}
-        {...refProps}
+        fieldRef={fieldRef}
```

Comment:
is it safe to do? there must have been a reason where we wouldn't want to pass the fieldRef prop. Remember this compoment is generic